### ISSUE TITLE
Echo filtering

### ISSUE THREAD

**Issue by theambient** [64characters/Telephone#28]
When you use telephone say on your iMac with loudspeakers caller hear some echo obviously his own voice which is irratable much. 

so i suggest to introduce some filtering of input adiou stream to reduce this echo and probably other noises. 

I think this at least echo could be implemented  with some dsp filters.

**Comment by eofster** [comment]
I believe it is much better now. Please tell me if it’s not and I’ll reopen the issue.

**Comment by mendelgordon** [comment]
@eofster This is an issue for me

**Comment by saibotma** [comment]
Yeah, there is still echo. I think this issue should be reopened.
